spec/models/emoji_spec.rb

```diff
@@ -87,6 +87,37 @@
     end
   end
 
+  describe 'version updates do' do
+    it 'should correct cache when global is stale' do
+      Emoji.global_emoji_cache["blonde_man"] = [
+        "invalid",
+        Emoji.new
+      ]
```

Comment:
It looks like we're not clearing this from the cache after this test. Perhaps we need a `after { Emoji.clear_cache }`?

Reply:
Ah calling `Emoji[":blonde_man:t3"]` will actually ensure the cache is correct. :thinking: I still think it'll be better if all tests can start from a blank cache though.

Reply:
yeah I could go either way here, but it looks like lots of other tests are warming up the cache and not clearing